docker/Dockerfile_nginx_develop

```diff
@@ -0,0 +1,13 @@
+# Stage 1: Get build artifacts from intelowl-ng
```

Comment:
Instead of a new file, we should use a docker variable `${INTELOWL_NG_DOCKER_TAG}` which can be defined in `docker/.env` file and then in the `test.override.yml` compose file, we just re-define it to `develop`.

Reply:
I tried that but it did not work. I had to use ARG to make it work properly in the FROM. Result is the same